src/axom/quest/SignedDistance.hpp

```diff
@@ -267,15 +267,47 @@ class SignedDistance
    */
   double computeDistance(const PointType& queryPnt) const;
 
+  /*!
+   * \brief Computes the distance of the given point to the surface mesh
+   * This overload also returns the computed closest point on the surface to the query point
+   * and the surface normal at that point
+   *
+   * \param [in] queryPnt user-supplied point.
+   * \param [out] closestPnt the closest point on the surface to \a queryPnt
+   * \param [out] surfaceNormal the surface normal of \a closestPt
+   *
+   * \note When the input is not a closed surface mesh, the assumption is that
+   *  the surface mesh divides the computational mesh domain into two regions.
+   *  Hence, the surface mesh has to span the entire domain of interest, e.g.,
+   *  the computational mesh at which the signed distance field is evaluated,
+   *  along some plane.
+   *
+   * \warning The sign of the distance from a given query point is determined by
+   *  a pseudo-normal which is computed at the closest point on the surface
+   *  mesh. For a non-watertight mesh, the sign of the distance is not defined
+   *  everywhere. Specifically, the sign is ambiguous for all points for which
+   *  a normal projection onto the surface does not exist.
+   *
+   * \return minDist the signed minimum distance to the surface mesh.
```

Comment:
```suggestion
   * \return the signed minimum distance to the surface mesh.
```

The return value is not a named quantity.